after_party.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: splash
+title: BSides Orlando 2025 After Party
+permalink: /after_party/
+---
+
+# After Party!
+
+Join us after the Conference to unwind and hang out! There will be board games, cash bar, music, and awesome people to talk to! The party starts at 7 pm and ends at 10, down at the Celeste Ballroom. Just go down University Blvd and turn left on Alafaya Trail, and onto Strategy Blvd, and you're there!
+
+Links to [Apple Maps](https://maps.apple.com/place?place-id=I1F6079BF27EAAE75&address=4105+N+Alafaya+Trail%2C+Orlando%2C+FL+32826%2C+United+States) and to [Google Maps](https://www.google.com/maps/search/?api=1&query=The+Celeste+Hotel,+Orlando,+a+Tribute+Portfolio+Hotel) can be found here.
```

Comment:
*I had my minions hand draw maps to help you find your way to the party. You have your choice of [Apple flavored](https://maps.apple.com/place?place-id=I1F6079BF27EAAE75&address=4105+N+Alafaya+Trail%2C+Orlando%2C+FL+32826%2C+United+States) or [Google flavored](https://www.google.com/maps/search/?api=1&query=The+Celeste+Hotel,+Orlando,+a+Tribute+Portfolio+Hotel). I told them we needed fish flavored but they didn't listen so good. Stupid minions.*